demos/dotnet/dotnet/dotnet.cs

```diff
@@ -78,9 +83,9 @@ static void Clean(string[] args, Log log)
         Directory.Delete(properties.PackagesDirectory, true);
     }
 
-    static void Build(string[] args, Log log)
+    static void Build(string[] args, Log log, bool buildLib = false)
```

Comment:
The parameter should be called "buildLibrary" or "buildDll". We try to avoid abbreviations in identifiers.